packages/app/README.md

```diff
@@ -1,20 +1,35 @@
-# `app`
+# `Öppna Skolplattformen App
 
-This is a demo app for using the API
+This is the app for Öppna Skolplattformen
 
-## Usage
+## Get started
 
-Clone the repo and make sure you have react-native installed:
+To get started using the iOS simulator start by installing the JavaScript
+dependencies using
 
-    npm i -g react-native
+```
+$ yarn
```

Comment:
I'd suggest to use `npx yarn` here, that would make this work if the user doesnt have yarn installed locally.